src/test/javascript/spec/component/admin/admin-feature-toggle.component.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { AdminFeatureToggleComponent } from 'app/admin/features/admin-feature-toggle.component';
+import { ArtemisTestModule } from '../../test.module';
+import { FeatureToggleService } from 'app/shared/feature-toggle/feature-toggle.service';
+import { MockFeatureToggleService } from '../../helpers/mocks/service/mock-feature-toggle.service';
+
+describe('AdminFeatureToggleComponentTest', () => {
+    let fixture: ComponentFixture<AdminFeatureToggleComponent>;
+    let comp: AdminFeatureToggleComponent;
+    let service: FeatureToggleService;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [ArtemisTestModule],
+            providers: [{ provide: FeatureToggleService, useClass: MockFeatureToggleService }],
+        })
+            .compileComponents()
+            .then(() => {
+                fixture = TestBed.createComponent(AdminFeatureToggleComponent);
+                comp = fixture.componentInstance;
+                service = TestBed.inject(FeatureToggleService);
+            });
+    });
+
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+
+    it('constructor', () => {
```

Comment:
why do you create a new AdminFeatureToggleComponent here instead of using comp. the constructor would also be called on TestBed.createComponent in the beforeEach hook.

Reply:
Thanks for the feedback, yes that's right. 

I was just trying to improve the coverage of the "% Branch " (75%). In the coverage result, it is suggested that line 31 (constructor) is not covered in the test and the code you've highlighted doesn't improve the result either. I will update it. 😊 

File                                                                                           | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
admin-feature-toggle.component.ts                                                  |     100 |       75 |     100 |     100 | 31